src/ir/enum_ty.rs

```diff
@@ -71,14 +71,18 @@ impl Enum {
         };
 
         declaration.visit(|cursor| {
-            if cursor.kind() == CXCursor_EnumConstantDecl {
+            let val = if is_signed {
+                cursor.enum_val_signed().map(EnumVariantValue::Signed)
+            } else {
+                if cursor.kind() == CXCursor_EnumConstantDecl {
```

Comment:
I wanted to keep it visually separate, because the inner if/else block can go away after #128 lands.